rest-service/manager_rest/storage/resource_models.py

```diff
@@ -1078,6 +1078,16 @@ class BlueprintsFilter(FilterBase):
     )
 
 
+def evaluate_workflow_parameters(deployment_id: str, parameters: dict):
+    # Keep this import line here because of circular dependencies
+    from manager_rest.dsl_functions import evaluate_intrinsic_functions
+
+    to_evaluate = {k: v for k, v in parameters.items()
+                   if isinstance(v, dict)}
```

Comment:
the `isinstance` like this would omit eg `parameter: [{'get_secret': 'x'}, {'get_secret': 'y'}]`, which is otherwise valid